rcamp/accounts/templates/account-request-create.html

```diff
@@ -93,7 +93,7 @@ <h4 class="modal-title">Definitions &amp; Form Help</h4>
         {% endif %}
             <div class="col-sm-12">
               <p id="ucb-login-help"><i class="fa fa-info-circle" aria-hidden="true"></i> UCB users use their CU IdentiKey username and password.</p>
-              <p id="csu-login-help"><i class="fa fa-info-circle" aria-hidden="true"></i> CSU users use their CSU EID as username. Password is <code>${csu_password},${duo_otp}</code> or <code>${csu_password},push</code>.</p>
+              <p id="csu-login-help"><i class="fa fa-info-circle" aria-hidden="true"></i> CSU users use their CSU EID as username. Password is <code>csu_password,duo_key</code> or <code>csu_password,push</code>.</p>
```

Comment:
While I generally agree, and would prefer to retain standard variable substitution syntax, the `csu_password,duo_key` and `csu_password,push` syntax matches what CSU uses in their internal documentation, and was explicitly requested; so we should stick with that.